Formula/knot-resolver.rb

```diff
@@ -26,15 +27,19 @@ class KnotResolver < Formula
   depends_on "hiredis" => :optional
   depends_on "libmemcached" => :optional
 
+  patch :DATA
+
   def install
     %w[all check lib-install daemon-install modules-install].each do |target|
-      system "make", target, "PREFIX=#{prefix}"
+      system "make", target, "PREFIX=#{prefix}", "ETCDIR=#{etc}/kresd"
```

Comment:
`, "ETCDIR=#{etc}/kresd"` should not be needed.

Reply:
I think it does, as it's hardcoded in the binary when looking for existing root.hints:
```
$ strings /usr/local/sbin/kresd | grep /etc/
/usr/local/etc/kresd/root.hints
```
Previously this was checking `/usr/local/Cellar/.../` path.

The right was would be to have an canonical root.hints and root.keys reused for all DNS servers (similar for what Debian/Ubuntu has in dns-root-data package), but I am not the guy for this kind of job in Homebrew.